src/dist.rs

```diff
@@ -315,13 +335,58 @@ impl Dist {
     }
 }
 
+#[cfg(feature = "sass")]
+fn sass(
+    static_dir: &std::path::Path,
+    dist_dir: &std::path::Path,
+    options: sass_rs::Options
+) -> Result<()> {
+    fn is_sass(path: &std::path::Path) -> bool {
+        matches!(
+            path.extension().map(|x| x.to_str()).flatten(),
+            Some("sass") | Some("scss")
+        )
+    }
+
+    fn should_ignore(path: &std::path::Path) -> bool {
+        path
+            .file_name()
+            .expect("static file ends by `..`")
+            .to_str()
+            .map(|x| x.starts_with("_"))
+            .unwrap_or(false)
+    }
+
+    log::trace!("Generating dist artifacts");
+    let walker = walkdir::WalkDir::new(&static_dir).into_iter();
+    for entry in walker {
```

Comment:
I believe the `for` already does `.into_iter()` for you.
```suggestion
    let walker = walkdir::WalkDir::new(&static_dir);
    for entry in walker {
```